packages/rs-dapi-client/src/platform/get_documents.rs

```diff
@@ -0,0 +1,186 @@
+//! `GetDocuments*` requests.
+
+use dapi_grpc::platform::v0::{self as platform_proto, Proof, ResponseMetadata};
+use drive::query::{OrderClause, StartClause, WhereClause};
+
+use crate::{platform::IncompleteMessage, transport::TransportRequest, DapiRequest, Settings};
+
+fn to_proto_where_clause(
+    where_clause: WhereClause,
+) -> platform_proto::get_documents_request::WhereClause {
+    let mut value_cbor_bytes = Vec::new();
+    ciborium::into_writer(&where_clause.value, &mut value_cbor_bytes)
```

Comment:
I'm not sure we should use cbor here. Shall we use bincode or what do we use to serialize documents for state?

Reply:
I don't know, previously whole `where` clause was a platform value encoded in CBOR, now only a `value` part is platform value in CBOR, so I didn't change formats we use there